graph/src/data/query/error.rs

```diff
@@ -50,6 +50,8 @@ pub enum QueryExecutionError {
     Unimplemented(String),
     EnumCoercionError(Pos, String, q::Value, String, Vec<String>),
     ScalarCoercionError(Pos, String, q::Value, String),
+    TooComplex(u64, u64), // (complexity, max_complexity)
+    TooDeep,
```

Comment:
Could we add the actual and allowed depth as arguments here, like with `TooComplex(u64, u64)`?

Reply:
The actual doesn't make sense because we stop if it hits the max. But I added the max to this error.

Reply:
Oh yeah, of course. Cool!